packages/charting/.storybook/preview.js

```diff
@@ -0,0 +1,30 @@
+import { initializeIcons } from '@uifabric/icons';
```

Comment:
Can we get rid of the duplication (especially this larger file) between the projects?

Reply:
I think I will refactor into a file inside /scripts and then call this logic (we still need preview.js physically present)

Reply:
Actually, i've already refactored everything I wanted to refactor into the generateStoriesFromExample() function. I kept the duplication for these:

1. addons should be customizable according to the needs of the package itself
2. require.context should to be called in that exact file for relative import reasons
3. some package in the future might not need to initializeIcons() so I'm leaving it there.

So, actually a little duplication allows for good amount of flexibility. The meat of the logic was to convert our .Example.js to stories anyway, so that's already factored into a separated place. The rest of the code is simply config.

Reply:
Okay. In either case you should really add the new files to the create-package script (and maybe delete anything we don't need any more while you're at it).